packages/engine/apache-arrow-js-bundle/README.md

```diff
@@ -13,7 +13,8 @@ The bundle should appear as `./dist/apache-arrow-bundle.js`
 
```

Comment:
Is there a reason we're not addressing the rest of [Upgrade JS arrow version](https://app.asana.com/0/0/1202244701819104/f) in this PR? Is there another PR to address it?

Reply:
I wasn't sure what that required.
The [3] will be removed, is the entire `setValid` method no longer needed?

Reply:
setValid was a patch which should have been released in the 8.0.0 version of Arrow (JS). So can we upgrade the arrow dependency and regenerate the bundle as a part of this PR and see if tests pass? (Without changing `setValid`)

Reply:
All tests seem to pass, I'll update the PR's description.